main.go

```diff
@@ -254,6 +257,19 @@ func main() {
 		envString("", "GITSYNC_ASKPASS_URL", "GIT_SYNC_ASKPASS_URL", "GIT_ASKPASS_URL"),
 		"a URL to query for git credentials (username=<value> and password=<value>)")
 
+	flGithubAPIURL := pflag.String("github-api-url",
+		envString("api.github.com", "GITHUB_API_URL"),
```

Comment:
Is this env name special (e.g. used somewhere deep in a library)?  I don't think so, right?

All the othe env vars are named "GITSYNC_xyz", so these should probably be "GITSYNC_GITHUB_APP_xyz"

Reply:
Done